frontend/src/components/homepage/StoryCard.tsx

```diff
@@ -119,6 +121,14 @@ const StoryCard = ({
   };
 
   const previewBook = () => history.push("/preview");
+
+  const primaryBtnText = () => {
+    if (displayMyStories) {
+      return storyTranslationId ? "review book" : "translate book";
+    }
+    return "view translation";
```

Comment:
if the user presses a button with view translation should that also try to assign them as either a reviewer/translator?

Reply:
Hmm. I think some of our existing logic is flawed here. 
If the user is browsing stories/translations, the call-to-action-button (primary button) should assign them as translator/reviewer.
If the user is looking at story translations they are already translators/reviewers of, the call-to-action-button (primary button) should open the translation platform. 

logic is now changed to reflect this.

Reply:
when we toggle for role "For translation", should only the "translate book" button show up? or can the "review book" button show up as well?
right now when i'm toggled to For Translation i see only the review book button, but when i toggle to For Review i see nothing (could also be my test data so i'll keep testing but wanted to clarify)